Assets/Scripts/Game/Questing/Person.cs

```diff
@@ -620,44 +627,26 @@ void AssignHomeTown()
                 }
             }
 
-            // For other NPCs use default scope and building type
-            Place.Scopes scope = Place.Scopes.Remote;
-            string buildingTypeString = houseString;
+            // For other NPCs use the given scope if any, else generate it at random
+            if (string.IsNullOrEmpty(scopeString))
```

Comment:
Should we treat Questor as always local?

Reply:
Questor is mostly local except when you get the quest through a letter as with the main quest or vampire quests